tuna/rocmlir/rocmlir_worker.py

```diff
@@ -47,25 +49,34 @@ class RocMLIRWorker(WorkerInterface):
   def __init__(self, **kwargs):
     """Constructor"""
     self.dbt = None
+    self.config_type = kwargs['config_type']
     super().__init__(**kwargs)
-    self.set_db_tables()
+    #    self.set_db_tables()
     self.result_attr = [column.name for column in inspect(self.dbt.results).c]
     self.result_attr.remove("insert_ts")
     self.result_attr.remove("update_ts")
 
+
+# Can either have one of these, or --device below, but no combinations.
```

Comment:
I had been using --device for a while, to specify the GPU a job is to run on.  A Teams discussion brought up the two environment variables and I thought they might work better or be easier.  Turns out they all behave the same, to the level that I can tell, and they also combine in some way when more than one is present, causing errors.

Reply:
My observation is the same, better to pick one of these methods, and not overlay them.